### PR TITLE
make use of the rollout option when promoting from a track to the sta…

### PR DESCRIPTION
**Pull request by belawaeckerlig**
Thanks for contributing to _fastlane_! Before you submit your pull request, please make sure to check the following boxes:
- [x] Run `rspec` for all tools you modified
- [x] Run `rubocop -a` to ensure the code style is valid
- [x] We currently don't accept new actions, please publish a plugin instead, more information in [Plugins.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Plugins.md)

Fixes https://github.com/fastlane/fastlane/issues/6501

### CODE REVIEW

**Comment by belawaeckerlig** [comment]
Please don't commit until it was discussed in #6501

**Comment by belawaeckerlig** [comment]
@asfalcone PR ready for review.
I changed the description of the rollout option, its default value and the verify block

File: supply/lib/supply/uploader.rb [hunk 1]
```diff
@@ -34,9 +34,9 @@ def perform_upload
 
     def promote_track
       version_codes = client.track_version_codes(Supply.config[:track])
-      client.update_track(Supply.config[:track], 1.0, nil)
+      client.update_track(Supply.config[:track], 0.1, nil)
```
Comment (asfalcone) [inline supply/lib/supply/uploader.rb]: Should this also use `Supply.config[:rollout]` for consistency?

Comment (belawaeckerlig) [inline supply/lib/supply/uploader.rb]: I wasn't sure about that. the value here does not really matter as long as it is between 0,05 and 0.5. But if we put there the value the user provides, it looks like the value matters. So I simply put some fixed value there.
But with a fixed value it is some kind of "magic" number...
So if you want me to change it, no problem. I'm fine with both ;)
I could also add a comment explaining that it does not matter which value is passed there

Comment (asfalcone) [inline supply/lib/supply/uploader.rb]: A comment is definitely important, probably more so than the actual value.

Please add a great comment and I'll get this merged. 👍

Comment (belawaeckerlig) [inline supply/lib/supply/uploader.rb]: @asfalcone sorry for the late reply. I added a comment

**Review by asfalcone — APPROVED** [review]

**Comment by asfalcone** [comment]
Thanks @belawaeckerlig! 🚀

**Pull request merged**